lib/hubspot/company.rb

```diff
@@ -111,6 +138,32 @@ def add_contact!(company_vid, contact_vid)
                                      },
                                      body: nil)
       end
+
+      # Updates the properties of companies
+      # NOTE: Up to 100 companies can be updated in a single request. There is no limit to the number of properties that can be updated per company.
+      # {https://developers.hubspot.com/docs/methods/companies/batch-update-companies}
+      # Returns a 202 Accepted response on success.
+      def batch_update!(companies)
```

Comment:
I think this is an artifact of merging and conflict resoution but there now appears to be two copies of `batch_update!`.